apps/betterangels-backend/clients/types.py

```diff
@@ -124,11 +127,19 @@ def search_client(
         """
         Returns client profiles with exact match on all provided search fields (case insensitive).
         All search fields are optional.
+
+        Accepts an excluded_client_profile_id param, to prevent the client profile being edited
+        from being flagged as a duplicate entry.
```

Comment:
I'd suggest removing references to duplicates as this is just a generic search. 
Maybe something like this is enough: `Accepts an excluded_client_profile_id param to exclude from result set.`